web3.nimble

```diff
@@ -16,6 +16,7 @@ requires "json_serialization"
 requires "nimcrypto"
 requires "stew"
 requires "stint"
+requires "contractabi >= 0.4.0 & < 0.5.0"
```

Comment:
the nimble resolver is unable to deal with version ranges correctly - we don't use them because they eventually cause the CI of unrelated projects to fail - ie depending on "latest" is the lesser evil